atlasdb-backup/src/test/java/com/palantir/atlasdb/backup/AtlasBackupServiceTest.java

```diff
@@ -116,6 +128,48 @@ public void prepareBackupThrowsIfNamespacesCollideWithExistingInProgressBackups(
                 .hasMessageContaining("Duplicated namespaces");
     }
 
+    @Test
+    public void cleanupBackupCloesLockRefresher() {
+        Set<InProgressBackupToken> tokens = ImmutableSet.of(IN_PROGRESS);
+        when(atlasBackupClient.prepareBackup(
+                        authHeader, PrepareBackupRequest.of(ImmutableSet.of(NAMESPACE, OTHER_NAMESPACE))))
+                .thenReturn(PrepareBackupResponse.of(tokens));
+
+        atlasBackupService.prepareBackup(ImmutableSet.of(ATLAS_SERVICE, OTHER_ATLAS_SERVICE));
+
+        atlasBackupService.cleanupBackup();
+        verify(lockRefresher).unregisterLocks(argThat(collection -> collection.contains(IN_PROGRESS)));
```

Comment:
Can we be more precise on this verify? Shouldn't this just be `tokens`?

Reply:
Frustratingly the tests failed if I did that, unless I wrapped `tokens.values()` with `ImmutableSet.copyOf(...)`.

I decided to make the tests a little dirtier instead of adding the set-copy in prod code.

Reply:
I guess I could make `tokens` the `values()` of some map, and then do `map.values()` here, but that seems strange.

Reply:
That sucks :/

Reply:
It's because `values` isn't a `Set`, it's some kind of `ConcurrentHashMapView` malarkey.